internal/dataplane/parser/parser.go

```diff
@@ -648,20 +656,44 @@ func getServiceEndpoints(
 	return targetsForEndpoints(endpoints)
 }
 
+// getIngressClassParametersOrDefault returns the parameters for the current ingress class.
+// If the cluster operators have specified a set of parameters explicitly, it returns those.
+// Otherwise, it returns a default set of parameters.
```

Comment:
Just wanted to say it's not required, but I do generally really appreciate when we take the time to put godoc on private functions, even when we think the function might be self-explanatory :+1: